components/infobox/wikis/formula1/infobox_unit_car.lua

```diff
@@ -19,62 +20,53 @@ local Cell = Widgets.Cell
 local Chronology = Widgets.Chronology
 local Title = Widgets.Title
 
-local _args
-
-local CustomUnit = Class.new()
-
+---@class Formula1UnitInfobox: UnitInfobox
```

Comment:
important to have different class between the 2 unit infoboxes so that annotations do not throw warnings
```suggestion
---@class Formula1CarInfobox: UnitInfobox
```